make install does not work.

make install does not work since there is no recipe for 'install' in the Makefile. Running make install produces the following output:

```
make install
make: *** No rule to make target `install'.  Stop.
```

--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,7 @@
 CC = gcc
 CFLAGS = -Wall -g -std=gnu11
 
+INSTALL_PATH = /usr/local/bin
 BIN = nuke
 
 OBJS = $(patsubst src/%.c,src/%.o,$(wildcard src/*.c))
@@ -19,6 +20,13 @@ src/%.o: src/%.c src/%.h
 gui ui: $(BIN)
 	pyuic5 -x $(UI) -o $(PY)
 
+install:
+	install -m 755 $(BIN) $(INSTALL_PATH)/$(BIN)
+
+uninstall:
+	rm -f $(INSTALL_PATH)/$(BIN)
+
 clean:
+	rm -f $(BIN)
 	rm -f src/*.o
 	rm -f $(PY)

--- a/README.md
+++ b/README.md
@@ -49,4 +49,4 @@ Run `sudo make install` to make and install Nuke.
 
 ## Uninstall
 
-Run `sudo make uninstall` to make and install Nuke.
+Run `sudo make uninstall` to make and uninstall Nuke.

--- a/README.md
+++ b/README.md
@@ -49,4 +49,4 @@ Run `sudo make install` to make and install Nuke.
 
 ## Uninstall
 
-Run `sudo make uninstall` to make and uninstall Nuke.
+Run `sudo make uninstall` to uninstall Nuke.

--- a/README.md
+++ b/README.md
@@ -45,4 +45,8 @@ NOTE: At the moment, Nuke can only be installed and run on Linux.
 
 ## Install
 
-Run `make install` to make and install Nuke.
+Run `sudo make install` to make and install Nuke.
+
+## Uninstall
+
+Run `sudo make uninstall` to make and install Nuke.

